index.html

```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+   <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane"></div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  <article class="card">
+  <div class="sane">
+  </div>
+  <div class="content">
+  <h1>Multi-post Stories <br>Gain + Glory</h1>
+  <ul class="category">
+  <li><a href="#">Ruby on Rails</a></li>
+  <li><a href="#">Css</a></li>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Html</a></li>
+  </ul>
+  <div class="open"><a href="#">See Project</a></div>
+  </div>
+  </article>
+  </div>
+</section>
+
+<section class="about">
+  <h1>About</h1>
+  <div class="about_info">
+  <p> I can help you build a product , 
+  feature or website Look through some 
+  of my work and experience! If you like 
+  what you see and have a project you need coded, 
+  don’t hestiate to contact me.</p>
+  </div>
+  <div class="resume"><a href="#">Get My Resume</a></div>
+  <ul class="skill">
+  <li class="weight">
+  <div class="featured language"></div>
+  <h1>Languages</h1>
+  <div class="cores"><ul>
+  <li><a href="#">Javascript</a></li>
+  <li><a href="#">Ruby</a></li>
+  <li><a href="#">Html</a></li>
+  <li><a href="#">Css</a></li>
+  </ul>
+  </div>
+  </li>
+  <li class="weight">
+  <div class="featured frame-works"></div>
+  <h1>Frameworks</h1>
```

Comment:
- [ ] Please the technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>`, each of them in a `<li>`